src/packages/core/components/icon/icon.element.ts

```diff
@@ -16,29 +16,48 @@ export class UmbIconElement extends UmbLitElement {
 	@state()
 	private _color?: string;
 
+	@state()
+	private _fallbackColor?: string;
+
+	/**
+	 * Color alias or a color code directly.\
+	 * If a color has been set via the name property, this property will override it.
+	 * */
 	@property({ type: String })
 	public set color(value: string) {
-		if (!value) return;
-		this.#setColorStyle(value);
+		if (value) {
+			this.#setColorStyle(value, true);
+		} else {
+			this._color = undefined;
```

Comment:
I think you don't need to explicitly set this to undefined if there is no value. The undefined value is the default of `_color` anyway.